index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Building Landing Page</title>
+    <meta charset="UTF-8" />
+    <<<<<<< HEAD
+    <link rel="stylesheet" href="./css/style.css" />
+    =======
+    <link rel="stylesheet" href="styles.css" />
+    >>>>>>> 09c5a842983cb9dfb9cdfe5ffabb2f75e70671b1
+    <script
+      src="https://kit.fontawesome.com/136b6c674f.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="fccp3.js"></script>
+  </head>
+  <body>
+    <header id="header">
+      <div class="logo">
+        <img
+          id="header-img"
+          src="https://freelogocreator.com/images/logo-large.png"
+        />
+      </div>
+      <nav id="nav-bar">
+        <a href="#features" class="nav-link">Features</a>
+        <a href="#video" class="nav-link">How It Works</a>
+        <a href="#pricing" class="nav-link">Pricing</a>
+      </nav>
+    </header>
+
+    <section class="container">
+      <form
+        id="form"
+        action="https://www.freecodecamp.com/email-submit"
+        method="POST"
+      >
+        <h1>Check your account</h1>
+        <input
+          type="email"
+          placeholder="Enter your email"
+          required
+          id="email"
+        />
+        <button id="submit" class="Enter-btn">Let's Go</button>
+      </form>
```

Comment:
- It would be better for the form to use a ul tag to group, and li per input 🙏
using the ul and li tags in HTML can improve the organization, structure, styling, and semantics of your content, making it more accessible, user-friendly, and maintainable.